### ISSUE TITLE
DotNetAdaptor fails to be restored

### ISSUE THREAD

**Issue by PeterUbimax** [microsoft/MixedRealityToolkit-Unity#7232]
## Describe the bug
The latest update to DotNetAdaptor seems to have a dependency on System.Drawing.Primitives that does not get restored automatically when restoring the DotNetWinRT nuget package. This was causing my MRTK project to have compile errors on editor loading.

## To reproduce

1. Ensure the nuget cache of system.drawing.primitives does not exist before proceeding. This is usually located here: %userprofile%\.nuget\packages\system.drawing.primitives.
2. Open a existing MRTK project that has been set to build for hololens 2 i.e. UWP, .net standard api.... 
Note: If you need to create a project with the settings from step 1 you will have to close the project and repeat step 1 - 2 
Result:
The DotNetAdaptor project fails to restore with the following error:

%userprofile%\.nuget\packages\microsoft.windows.mixedreality.dotnetwinrt\0.5.1043\build\Microsoft.Windows.MixedReality.DotNetWinRT.targets(26,5): error MSB3030: Could not copy the file "%userprofile%\.nuget\packages\microsoft.windows.mixedreality.dotnetwinrt\0.5.1043\build\..\..\..\System.Drawing.Primitives\4.3.0\lib\netstandard1.1\System.Drawing.Primitives.dll" because it was not found. [C:\Development\MixedRealityToolkit-Unity\Assets\MixedRealityToolkit.Providers\WindowsMixedReality\DotNetAdapter\DotNetAdapter.csproj]
UnityEngine.Debug:LogError(Object)
Microsoft.Build.Unity.MSBuildProjectBuilder:LogProgressMessage(String, ProgressMessageType) (at Library/PackageCache/com.microsoft.msbuildforunity@0.8.3/Editor/ProjectBuilder/MSBuildProjectBuilder.cs:348)
Microsoft.Build.Unity.<>c__DisplayClass9_0:<BuildProjects>b__1(ValueTuple`2) (at Library/PackageCache/com.microsoft.msbuildforunity@0.8.3/Editor/ProjectBuilder/MSBuildProjectBuilder.cs:181)
Microsoft.Build.Unity.DelegateProgress`1:Report(ValueTuple`2) (at Library/PackageCache/com.microsoft.msbuildforunity@0.8.3/Editor/ProjectBuilder/MSBuildProjectBuilder.cs:489)
Microsoft.Build.Unity.<>c__DisplayClass10_0:<BuildProjectsAsync>b__1(ValueTuple`2) (at Library/PackageCache/com.microsoft.msbuildforunity@0.8.3/Editor/ProjectBuilder/MSBuildProjectBuilder.cs:293)
Microsoft.Build.Unity.DelegateProgress`1:Report(ValueTuple`2) (at Library/PackageCache/com.microsoft.msbuildforunity@0.8.3/Editor/ProjectBuilder/MSBuildProjectBuilder.cs:489)
Microsoft.Build.Unity.<>c__DisplayClass17_0:<ExecuteMSBuildAsync>b__0(Object, DataReceivedEventArgs) (at Library/PackageCache/com.microsoft.msbuildforunity@0.8.3/Editor/ProjectBuilder/MSBuildProjectBuilder.cs:435)
System.Threading._ThreadPoolWaitCallback:PerformWaitCallback()

## Expected behavior

The DotNetAdaptor should restore any dependencies it requires automatically.

## Your setup (please complete the following information)

- Unity Version 2018.4.14f1
- MRTK Version v2.2.0

## Target platform (please complete the following information)

- HoloLens 2

## Additional context

This offending nuget package is [Microsoft.Windows.MixedReality.DotNetWinRT](https://api.nuget.org/v3-flatcontainer/microsoft.windows.mixedreality.dotnetwinrt/0.5.1043/microsoft.windows.mixedreality.dotnetwinrt.0.5.1043.nupkg). Specifically the latest update "0.5.1043 "

**Comment by keveleigh** [comment]
@dfields-msft, is there a new dependency?

**Comment by dfields-msft** [comment]
The use of `System.Drawing.SizeF` was introduced in early December as part of the changes to enable the `HolographicDepthReprojectionMethod` API; I believe this predates MRTK 2.2's use of M.W.MR.DotNetWinRT. However, a package dependency on `System.Drawing.Primitives` was just added to the NuGet definition at the end of January as part of @majaeger's changes to support the .NET scripting backend.

**Comment by davidkline-ms** [comment]
I recently restored 0.5.1043 (after modifying the csproj file in mrtk) with no errors.

**Comment by davidkline-ms** [comment]
Could it be an issue with MSBuild for Unity 0.8.3 that has been fixed in 0.9.1?

**Comment by wiwei** [comment]
@PeterUbimax I can confirm I see an error in the editor, though it does resolve after the restore happens.

Do you get in a permanently busted state?

**Comment by majaeger** [comment]
@PeterUbimax, I've published an updated nuget package for Microsoft.Windows.MixedReality.DotNetWinRT, version 0.5.1044, which should fix the issue. 

Can you try updating to the newer version and let us know if that revolves the problem?

**Comment by PeterUbimax** [comment]
@wiwei It's not permanently busted and the workaround at least for me was to create a new mrtk project and switch the build player to WSA. Then the nuget cache for system.drawing,primitives gets downloaded and solves any issues with other projects.

**Comment by PeterUbimax** [comment]
@majaeger Can confirm the latest nuget packge for DotNetWinRT resolves the issue I was seeing. 
FYI: for the definitive way to test this I did the following:

1. Deleted any cache of system.drawing.primitives from the following location: %userprofile%.nuget\packages\system.drawing.primitives.

2. Download the MRTK project  and navigate to _MixedRealityToolkit.Providers\WindowsMixedReality\DotNetAdapter_

3. Ensure the ".obj" and "Plugins" folders are deleted

4. Run command "dotnet msbuild DotNetAdapter.csproj -restore  -v:minimal -p:NuGetInteractive=true  -t:Build -p:Configuration=Release"

Result: The system.drawing.primitives nuget package restored as expected

Note: if you want to see the original behavior I reported repeat the above steps but open the DotNetAdapter.csproj and replace line "<PackageReference Include="Microsoft.Windows.MixedReality.DotNetWinRT" Version="0.5.*" />" with "PackageReference Include="Microsoft.Windows.MixedReality.DotNetWinRT" Version="0.5.1043" />" before doing step 4.
